wi-fi/whd/whd_wifi_api.c

```diff
@@ -4093,3 +4093,24 @@ whd_tko_toggle(whd_interface_t ifp, whd_bool_t enable)
 	}
 	return result;
 }
+
+whd_result_t
+whd_print_wlan_log(whd_driver_t whd_drv)
```

Comment:
Coding style - retval should be in the same line as the rest of the function prototype

Reply:
Original code formatted by clang-format.

Reply:
That's weird - our clang-format config shouldn't output such results